### ISSUE TITLE
Have you gotten good results now?

### ISSUE THREAD

**Issue by ShengleiH** [whqwill/seq2seq-keyphrase-bert#2]
Hi, thanks for sharing your good try. I am also doing some attempts to use BERT in my generation task. Have you gotten any good results now? Or are there any papers recommended to read? Thanks a lot!!!!

**Comment by thenewcomer** [comment]
I meet a similar problem in my code, which is the normal two GRU encoders performance better than using BERT as encoders. The performance of BERT with GRU keeps relatively low. I think something must be wrong and I am checking my code. May I know have you got good results now?

**Comment by vnik18** [comment]
@thenewcomer Hi, I am facing a similar problem. The performance is dropping when I using BERT encoders. Please let me know if you managed to fix this problems and get good results now? Thank you.

**Comment by zhuang-li** [comment]
yes met the same issue. While I see some works report that they use BERT as encoder and the RNN as decoder and got good results. Don't know why.
